geonode/base/models.py

```diff
@@ -624,8 +625,8 @@ def set_latlon_bounds(self, box):
         Set the four bounds in lat lon projection
         """
         self.bbox_x0 = box[0]
-        self.bbox_x1 = box[1]
-        self.bbox_y0 = box[2]
+        self.bbox_y0 = box[1]
+        self.bbox_x1 = box[2]
```

Comment:
I even tried to screen it again the third time for separate PR in upstream. For us, we only use leaflet map view, so we know where to look.